cybersyn/scripts/central-planning.lua

```diff
@@ -652,13 +694,13 @@ local function tick_poll_station(map_data, mod_settings)
 						f, a = once, station.network_name
 					end
 					for network_name, _ in f, a do
-						local item_network_name = network_name..":"..item_name
+						local item_network_name = network_name .. ":" .. item_hash -- FIXME: item_name or item_hash?
 						local stations = all_r_stations[item_network_name]
 						if stations == nil then
 							stations = {}
 							all_r_stations[item_network_name] = stations
-							all_names[#all_names + 1] = item_network_name
-							all_names[#all_names + 1] = v.signal
+							all_names[#all_names + 1] = item_network_name	
+							all_names[#all_names + 1] = v.signal    -- FIXME: This is sus. Why is the same element immediately overwritten?
```

Comment:
This is a second entry (`#all_names` changed after the last line)

Reply:
Looks like I was too tired when I added this comment. Comment was removed in the merged commits.